fix(dcmjs): #231 Fix the normalize Alphabetic twice

wayfarer3130 opened the pull request:
Fixed issue 231 where normalizing twice fails, and prevents side affects for modifying the array of length one, but doesn't fix any of the issues modifying longer non-sequence arrays, nor modifications to the properties of the child objects.

pieper commented on test/test_data.js:
```diff
@@ -115,6 +115,14 @@ function makeOverlayBitmap({ width, height }) {
+        const datasets = JSON.parse(dicomJSON);
+        const natural0 = DicomMetaDictionary.naturalizeDataset(datasets[0]);
+        // Shouldn't throw an exception
+        const natural0b = DicomMetaDictionary.naturalizeDataset(datasets[0]);
```
Can you also test if they are the same?

wayfarer3130 replied:
Yes, done.

pieper reviewed (changes requested):
Can you add some more description in your commit message?  Why did the previous code lead to the issue, what changes did you make and why did they fix the problem?  It will help to have more context if this or a related issue comes up again.

wayfarer3130 commented:
> Can you add some more description in your commit message? Why did the previous code lead to the issue, what changes did you make and why did they fix the problem? It will help to have more context if this or a related issue comes up again.

Redid the commit message when I added the other fix and updated to master.

pieper reviewed (approved):
Thanks for the updates 👍
